### ISSUE TITLE
Is this project still alive? 

### ISSUE THREAD

**Issue by Aviral190694** [leavez/cocoapods-binary#130]
Thanks for your feedback!

If it's a bug:

- What's your environment:
  - run ` pod env | egrep 'CocoaPods |Xcode|cocoapods-binary' ` then paste the result here 
- A demo project is helpful:
  - It will be very helpful if provides a demo project that reproduce the problem. A blank project with cocoapods-binary can be downloaded [here](https://github.com/leavez/cocoapods-binary/files/3086250/BinaryDemo.zip), which can be remodeled to a minimal problematic environment. Attach the demo.zip to the comments.

This is a really neat project but need some work. Adding the issue to know if someone is working on it in a fork or something. If not I am looking forward to learn and collaborate on this as this is really cool project. 

Please let me know. 
@leavez any plans in case you have to work on this?

**Comment by muukii** [comment]
Yeah, this plugin is super awesome. it helps many developers for saving time.
I hope this repository works again.
In another way, it's might be good the owner of this repo gives someone permission to maintain.
Finally, the owner absolutely get sponsors.

**Comment by muukii** [comment]
So, what do you think about I create a forked repo?
And we handle PRs to maintain in there And also waiting for @leavez comes back.
I'll create a PR to merge all of the updates into this repo in the future.

**Comment by muukii** [comment]
For now, I created https://github.com/leavez/cocoapods-binary/pull/137

**Comment by swiftyfinch** [comment]
Hello! I've had used this great repo before, but I also had trouble and decided to write my own similar solution.
You can try this alternative 🏈 [Rugby](https://github.com/swiftyfinch/Rugby). Maybe it'll help you.
